src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,615 @@
  * limitations under the License.
  */
 
-export const human38stats = {
-  assembly_stats: {
-    chromosome_count: 23,
-    genome_size: 3096649726,
-    total_sequence_length: 4537931177
+import { SpeciesStatsSection } from 'src/content/app/species/state/general/speciesGeneralHelper';
+
+type RawSpeciesStats = {
+  [genomeId: string]: {
+    [key in SpeciesStatsSection]: {
+      [key: string]: string;
```

Comment:
This is something I would definitely urge we change. The type of field value can be at least one of the following: string, number, null